chef/cookbooks/hyperv/attributes/default.rb

```diff
@@ -84,6 +84,6 @@
 default[:service][:neutron][:name] = "neutron-hyperv-agent"
 default[:service][:neutron][:displayname] = "OpenStack Neutron Hyper-V Agent Service"
 default[:service][:neutron][:description] = "Service Wrapper for Openstack Neutron Hyper-V Agent - Manages the OpenStack Neutron Hyper-V Agent as a Windows Service"
-default[:service][:ceilometer][:name] = "ceilometer-compute-agent"
-default[:service][:ceilometer][:displayname] = "OpenStack Ceilometer Compute Agent Service"
-default[:service][:ceilometer][:description] = "Service Wrapper for Openstack Ceilometer Compute Agent - Manages the OpenStack Ceilometer Compute Agent as a Windows Service"
+default[:service][:ceilometer][:name] = "ceilometer-agent-notification"
+default[:service][:ceilometer][:displayname] = "OpenStack Ceilometer Agent Notification Service"
+default[:service][:ceilometer][:description] = "Service Wrapper for Openstack Ceilometer Agent Notification- Manages the OpenStack Ceilometer Compute Agent as a Windows Service"
```

Comment:
Metrics/LineLength: Line is too long. [177/150] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)